bot/resources/tags/string-formatting.md

```diff
@@ -0,0 +1,27 @@
+**String Formatting Mini Language**
+The String Formatting Language in Python is a powerful way to tailor the display of strings and other data structures. This string formatting mini language works for f-strings and .format().
+
+Take a look at some of these examples!
+```py
+my_num = 2134234523
+print(f"{my_num:,}")
```

Comment:
```suggestion
print(f"{my_num = :,}")
```

Reply:
I disagree with these, it means that we show no examples without it and now someone might think you have to do it.